action.yml

```
@@ -4,6 +4,9 @@ icon: calendar
 branding:
   icon: 'calendar'
   color: 'purple'
+inputs:
+  prefix:
+    description: 'String to prepend to the tag name'
```

Could it be that we also need to specify either `required: true` or a default value?

```suggestion
     description: 'String to prepend to the tag name' 
     default: '' 
```